tests/test_generate_index.py

```diff
@@ -224,16 +225,22 @@ def test_index_no_existing_releases(setup_test_env): #todo move these tests?
                                              "--module_path", "../test_package", "../another_test_package"])
     index_exists = run(["git", "ls-tree", "-d", f"origin/{target_branch}", "index.html"],
                                     capture_output=True, text=True, check=True)
-    assert index_exists.returncode == 0
     index_source_exists = run(["git", "ls-tree", "-d", f"origin/{target_branch}", "_sources/index_template.jinja"],
                                     capture_output=True, text=True, check=True)
-    assert index_source_exists.returncode == 0
 
     run(["git", "checkout", target_branch], check=True)
     with open("index.html") as index_file:
         index_content = index_file.readlines()
-    for i in range(0, len(correct_content)):
-        assert correct_content[i].strip() == index_content[i].strip()
+
+    stripped_correct_content = sorted([correct_content[i].strip()
+                                       for i in range(0, len(correct_content))])
+    stripped_index_content = sorted([index_content[i].strip()
+                                     for i in range(0, len(index_content))])
```

Comment:
Is the index even needed?
```suggestion
    stripped_index_content = sorted(
        (c.strip() for c in correct_content)
    )
```